webpack/config.react-native.js

```diff
@@ -0,0 +1,13 @@
+var path = require('path');
+
+var mergeDeep = require('./merge-deep');
+var sharedConfig = require('./config.shared');
+
+module.exports = mergeDeep(sharedConfig, {
+  output: {
+    library: "PusherPlatform",
+    libraryTarget:"commonjs2",
```

Comment:
It looks like these files are not picked up by Prettier

Reply:
A good spot, although this is a non-prettier-ed branch.

I'll fix it in the soon to be PRed prettier-ed + lint-ed branch